recipes/atlas-bash-util/build.sh

```diff
@@ -1,2 +1,2 @@
 mkdir -p $PREFIX/bin
-cp *.sh atlas-bash-util atlas-lsf $PREFIX/bin
+cp *.sh atlas-bash-util atlas-lsf atlas-slurm $PREFIX/bin
```

Comment:
```suggestion
cp *.sh atlas-bash-util atlas-lsf $PREFIX/bin
```

Reply:
We dont need to copy `atlas-slurm` anymore

Reply:
I think we have to do it, as it was done before for `atlas-lsf`

Reply:
ohh my bad.. I am wrong